### ISSUE TITLE
RICH Layout and prompt - terminal user interface

### ISSUE THREAD

**Issue by PrincesNunu** [prompt-toolkit/python-prompt-toolkit#1528]
Hello:
I am doing a small program of a user interface for running on a terminal. I use RICH to define Layouts and I need to put into one layout a prompt for asks inputs and show into another Layout. Did someone have make this before? It is possible? How?
Thanks in advance!

**Comment by jonathanslenders** [comment]
I don't think this has been done before. In theory it should be possible with some adapters, I guess, but that will be quite a bit of work to make that happen.
